opennms-doc/guide-install/src/asciidoc/text/opennms/introduction.adoc

```diff
@@ -72,20 +72,25 @@ TIP: Verify the release of _OpenNMS_ packages with `yum info opennms`.
 
 ==== Specific Release on Debian-based system
 
+Create a new apt source file (eg: `/etc/apt/sources.list.d/opennms.list`), and add the following 2 lines:
+
+.Package repository configuration for Debian-based systems
 [source, shell]
 ----
-vi /etc/apt/sources.list.d/opennms.list
+deb http://debian.opennms.eu ${RELEASE} main <1>
+deb-src http://debian.opennms.eu ${RELEASE} main <1>
 ----
 
-.Package repository configuration for RHEL-based systems
+<1> Replace `${RELEASE}` with a release name like `testing` or `snapshot`.
```

Comment:
It would be nice to have all available possibilities here or at a referenced article.